src/main/scripts/package.sh

```diff
@@ -53,6 +53,7 @@ if [[ "$platform" == "win" ]]; then
         "--java-options" "$jvmOptions --add-opens jdk.crypto.mscapi/sun.security.mscapi=ALL-UNNAMED"
         "--win-shortcut-prompt"
         "--win-menu"
+        "--add-launcher" "Autogram-cli=$resourcesDir/windows-cli-build.properties"
```

Comment:
Neviem aka je windows konvencia ale linux dava vsetko malym, cize `autogram-cli` ?

Reply:
Našiel som na nete všeobecne convention, že "no reason to use shift key". To znamená žiadne veľké písmená a žiadne podtržníky. Rozbehol som virtuálku Windows a nenašiel som tam veľmi iné.